### PR TITLE
Global ARP timer option

### LINKED ISSUE #477: Global ARP ageing knob
**Issue Type**

- Template enhancement
- Role enhancement

**Is your feature request related to a problem? Please describe.**
Instead of doing this per interface we should support global knob.

**Describe the solution you'd like**

```arp aging timeout default```

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by onurgashi**
## Change Summary

Knob to change ARP timer globally.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [x] Documentation content changes
- [ ] Other (please describe):

## Related Issue(s)

Fixes #477 

## Component(s) name
eos_cli_config_gen

## Proposed changes

Added knob
Added doc
Tested in DUT and molecule (wrong title for doc, fixed it in later commit.)

## How to test

Molecule

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://www.avd.sh/docs/contributing/) document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed ([`pre-commit`](https://www.avd.sh/docs/installation/development/#python-virtual-environment), `make linting` and `make sanity-lint`).
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly

### CODE REVIEW

**Review by titom73 — APPROVED** [review]
Looks good to me.

Just need to confirm with @aphillipps if the data model is OK

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/arp.j2 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% if arp is defined and arp is not none %}
+{%     if arp.aging is defined and arp.aging is not none %}
+{%         if arp.aging.timeout_default is defined and arp.aging.timeout_default is not none %}
+Global ARP timeout: {{ arp.aging.timeout_default }}
+{%         endif %}
+{%     endif %}
+{% else %}
+Global ARP timeout not defined.
+{% endif %}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/arp.j2]: Missing newline?

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/arp.j2 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{# eos - ARP settings #}
+{% if arp is defined and arp is not none %}
+{%     if arp.aging is defined and arp.aging is not none %}
+{%         if arp.aging.timeout_default is defined and arp.aging.timeout_default is not none %}
+!
+arp aging timeout default {{ arp.aging.timeout_default }}
+{%         endif %}
+{%     endif %}
+{% endif %}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/arp.j2]: Missing newline

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]
Missing newlines at the end of the new templates

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/arp.j2 (hunk 1 above)
Comment (onurgashi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/arp.j2]: Resolved, thank you!

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/arp.j2 (hunk 2 above)
Comment (onurgashi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/arp.j2]: Resolved, thank you!

**Comment by onurgashi** [comment]
Resolved the discrepancies.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Review by titom73 — APPROVED** [review]
LGTM!

**Pull request merged**